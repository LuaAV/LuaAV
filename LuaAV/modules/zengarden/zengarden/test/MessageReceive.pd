#N canvas 554 371 450 300 10;
#X obj 27 -67 bng 15 250 50 0 empty empty empty 17 7 0 10 -262144 -1
-1;
#X obj 66 -69 loadbang;
#X obj 175 -64 receive;
#X obj 175 -30 print;
#X obj 27 -34 t b b;
#X obj 174 36 print printed apple;
#X msg 70 7 hello;
#X obj 27 42 send;
#X msg 27 7 37;
#X obj 70 40 s apple;
#X obj 174 2 r apple;
#X connect 0 0 4 0;
#X connect 1 0 4 0;
#X connect 2 0 3 0;
#X connect 4 0 8 0;
#X connect 4 1 6 0;
#X connect 6 0 9 0;
#X connect 8 0 7 0;
#X connect 10 0 5 0;
