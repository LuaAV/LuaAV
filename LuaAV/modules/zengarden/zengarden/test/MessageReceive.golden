[@ 0.000ms] printed: hello
[@ 0.000ms] print: 37